specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkVirtualAppliance.json

```diff
@@ -0,0 +1,509 @@
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Delete successful."
+          },
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously."
+          },
+          "204": {
+            "description": "Request successful. Resource with the specified name does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete NetworkVirtualAppliance": {
+            "$ref": "./examples/NetworkVirtualApplianceDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      },
+      "get": {
+        "tags": [
+          "NetworkVirtualAppliances"
+        ],
+        "operationId": "NetworkVirtualAppliances_Get",
+        "description": "Gets the specified Network Virtual Appliance",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "networkVirtualApplianceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of Network Virtual Appliance"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Expands referenced resources."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a Network Virtual Appliance resource.",
+            "schema": {
+              "$ref": "#/definitions/NetworkVirtualAppliance"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get NetworkVirtualAppliance": {
+            "$ref": "./examples/NetworkVirtualApplianceGet.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "NetworkVirtualAppliances"
+        ],
+        "operationId": "NetworkVirtualAppliances_Update",
```

Comment:
```suggestion
        "operationId": "NetworkVirtualAppliances_UpdateTags",
```

Reply:
Done

Reply:
IMO just _update is reasonable, since RP can add more patchable properties to the payload

---
In reply to: [365040311](https://github.com/Azure/azure-rest-api-specs/pull/8010#discussion_r365040311) [](ancestors = 365040311)